### DISCUSSION TITLE
add utf8 support?

### DISCUSSION THREAD

**Discussion by lovecoding2023** [pyvista/pyvista#3697] [Q&A]
Dear All, 
I really want to add support for utf-8 to such a great visualization tool, like "add_text",  I tried the content mentioned in the following link, [https://github.com/pyvista/pyvista/discussions/2166](https://github.com/pyvista/pyvista/discussions/2166) I still can't solve it. Could you please give me some tips and examples on how to support utf-8 characters ? Thank you in advance :)

**Comment by tkoyama010** [comment]
Basically, PyVista's feature is vtk wrapped with Pythonic function. Therefore, the functionality of PyVista depends on VTK. vtk has a vtkUnicodeString class, which may be used to achieve this.

**Reply by lovecoding2023** [reply]
Thank you for your answer, which means that I can only compile the required VTK language version myself?
